include/adbregisters.h

```diff
@@ -1,3 +1,11 @@
+/**
+ * QuokkADB
+ * Copyright (C) 2022 Rabbit Hole Computing LLC
```

Comment:
This isn't how copyright works. You can't just take a file and claim it as your own.

Reply:
Please take a look at pull request #4 and let me know what you think